### ISSUE TITLE
8bits vectors?

### ISSUE THREAD

**Issue by delehef** [jackmott/simdeez#21]
Hi!

Is there a particular reason for which only 16- and 32-bits wide integer vectors are implemented, but not the 8-bits wide ones?

**Comment by jackmott** [comment]
nope, prs welcome. next thing i am working on is probably wasm support,
could add 8bit instructions eventually but might be a while

**Comment by rdaum** [comment]
@jackmott Not being too familiar with the simdeez source tree, but having gotten far enough last night to notice that I couldn't seem to be able to load an i8 vector (though 2.0 documentation is missing maybe? so I may have missed something), and if I were to want to help out to add it in a PR ... what's missing at this point? I see i8 support on tip of tree, but not clear if there's actually a way to create and load such a vector?
